hack/chains/deploy_local_rekor.tpl

```diff
@@ -0,0 +1,30 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: rekor
+spec:
+  destination:
+    name: ''
+    namespace: rekor-server
+    server: 'https://kubernetes.default.svc'
+  source:
+    path: helm-charts/rekor
+    repoURL: 'https://github.com/sigstore/sigstore-helm-operator'
+    targetRevision: HEAD
```

Comment:
I would have expected 'main' instead of HEAD, but I guess it would work the same, and maybe HEAD makes less assumptions, so not if there's any advantage.